packages/mobile/src/transactions/ExchangeFeedItem.tsx

```diff
@@ -59,32 +73,33 @@ export function ExchangeFeedItem(props: Props) {
   return (
     <Touchable onPress={onPress}>
       <View style={styles.container}>
-        {showImage && (
-          <View style={styles.imageContainer}>
-            <Image
-              source={
-                inCurrency === CURRENCY_ENUM.DOLLAR
-                  ? require(`src/transactions/ExchangeGreenGold.png`)
-                  : require(`src/transactions/ExchangeGoldGreen.png`)
-              }
-              style={styles.image}
-              resizeMode="contain"
-            />
-          </View>
-        )}
-        <View style={[styles.contentContainer, showImage && styles.contentContainerWithImage]}>
+        <View style={styles.imageContainer}>
+          <Image
+            source={
+              inCurrency === CURRENCY_ENUM.DOLLAR
+                ? require(`src/transactions/ExchangeGreenGold.png`)
+                : require(`src/transactions/ExchangeGoldGreen.png`)
+            }
+            style={styles.image}
+            resizeMode="contain"
+          />
+        </View>
+        <View style={styles.contentContainer}>
           <View style={styles.titleContainer}>
             <Text style={fontStyles.bodySmallSemiBold}>{t('exchange')}</Text>
             <Text
               style={[
-                dollarDirection === '-'
+                amountDirection === '-'
                   ? fontStyles.activityCurrencySent
-                  : fontStyles.activityCurrencyReceived,
+                  : {
+                      ...fontStyles.activityCurrencyReceived,
+                      color: showGoldAmount ? colors.celoGold : colors.celoGreen,
+                    },
                 styles.amount,
               ]}
             >
-              {dollarDirection}
-              {getMoneyDisplayValue(dollarAmount)}
+              {amountDirection}
```

Comment:
Should we include this as a param for `getMoneyDisplayValue` and let it add the amount direction as necessary?

Reply:
I'd say let's leave this for now and reconsider if we see this pattern repeat.